src/views/Farms/components/FarmCard/CardActionsContainer.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useMemo, useState, useCallback } from 'react'
+import BigNumber from 'bignumber.js'
+import styled from 'styled-components'
+import { useWallet } from '@binance-chain/bsc-use-wallet'
+import { provider } from 'web3-core'
+import { getContract } from 'utils/erc20'
+import { Button, Flex, Text } from '@pancakeswap-libs/uikit'
+import { Farm } from 'state/types'
+import { useFarmFromSymbol, useFarmUser } from 'state/hooks'
+import useI18n from 'hooks/useI18n'
+import UnlockButton from 'components/UnlockButton'
+import { useApprove } from 'hooks/useApprove'
+import StakeAction from './StakeAction'
+import HarvestAction from './HarvestAction'
+
+const Action = styled.div`
+  padding-top: 16px;
+`
```

Comment:
We can use `<Text pt="16px" />`.

Makes me think we might need a `<Box />` component or rename the text one.

Reply:
Didn't realise `Text` is just a wrapper around a div. Still... this feels like accepting horrible, un-semantic naming, just to save on declaring this styled component? Not sure I see it as an improvement. Agree on the `<Box />` uikit idea 👍

Reply:
>Still... this feels like accepting horrible, un-semantic naming, just to save on declaring this styled component? 
You know you are right, let's wait for the box component and keep this as-is.